src/ssh-util/src/tunnel_manager.rs

```diff
@@ -148,11 +149,11 @@ impl SshTunnelManager {
 }
 
 /// Identifies a connection to a remote host via an SSH tunnel.
-#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord)]
-struct SshTunnelKey {
-    config: SshTunnelConfig,
-    remote_host: String,
-    remote_port: u16,
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct SshTunnelKey {
+    pub config: SshTunnelConfig,
+    pub remote_host: String,
+    pub remote_port: u16,
 }
```

Comment:
This looks left over from an earlier commit. At least, we had a discussion elsewhere about _not_ making this public API. As a general rule, especially for these utility crates where it's easy to come up with well defined API boundaries, we should take a lot of care in ensuring that we expose well documented and intentional public API, rather than just exposing what's convenient!